driver/collector/collector_factory.py

```diff
@@ -263,17 +270,24 @@ def get_collector(
             collector = MysqlCollector(conn, version)
         elif driver_conf["db_type"] in ["postgres", "aurora_postgresql"]:
             pg_conf = create_db_config_postgres(driver_conf)
-            conns: Dict[str, Any] = {}
-            if driver_conf['postgres_db_list'] is not None:
+            if driver_conf["postgres_db_list"] is not None:
```

Comment:
I add new code to handle cloudformation input after you signed off, could you take a look at this as well? No rush wait until tomorrow :) @YuzeLiao